src/librbd/cache/rwl/Types.h

```diff
@@ -4,8 +4,13 @@
 #ifndef CEPH_LIBRBD_CACHE_RWL_TYPES_H
 #define CEPH_LIBRBD_CACHE_RWL_TYPES_H
 
+#include "acconfig.h"
```

Comment:
Nit: perhaps split up the types between generic, RWL-specific, and SSD-specific so you don't need to keep `#ifdef`-ing individual things